test/support/tester/utils.ts

```diff
@@ -14,7 +13,7 @@ export function isNullOrUndefined<T>(value: T | null | undefined): boolean {
   return typeof value === 'undefined' || value === null;
 }
 
-export function stubMessageChannel(t: TestContext & Context<any>) {
```

Comment:
Possibly renamed to [ExecutionContext](https://github.com/avajs/ava/blob/f06c506d327e89800d9760c678ab0feb78f1a93d/index.d.ts#L315) ?

Reply:
Can confirm this was renamed to `ExecutionContext`